.github/workflows/dbt-checks.yaml

```
@@ -0,0 +1,57 @@
+          python-version: "3.9"
+
+      - name: Install requirements
+        run: |-
+          pip install --no-cache-dir -r requirements.txt
+
+      - name: Install sqlfmt
+        run: pip install shandy-sqlfmt[jinjafmt]
+
+      # Setup gcloud CLI
+      - name: Setup Google Cloud CLI
+        uses: google-github-actions/setup-gcloud@v1
```

actionlint (1.7.7)

the runner of "google-github-actions/setup-gcloud@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)